.gitignore

```diff
@@ -3,4 +3,3 @@
 /coverage.clover
 /phpunit.xml
 /.idea
-/composer.lock
```

Comment:
Why do you want to add the `composer.lock`? Composer does not check it when installing the library as a dependency.

Reply:
Because this way other devs will get the same version as everyone else. It is also recommended practice: https://getcomposer.org/doc/01-basic-usage.md#commit-your-composer-lock-file-to-version-control

It's important to have a reproducible dev environment.

Reply:
I don't think that that comment applies to libraries. See those project for example:

* https://github.com/symfony/symfony
* https://github.com/Seldaek/monolog
* https://github.com/guzzle/guzzle
* https://github.com/sebastianbergmann/phpunit

In my opinion this gives developers a false sense on how this library is used.

Reply:
I still think it helps to make sure `composer install` will always produce the same result, but if you do not want to include it, it's fine with me. It is true that for a library it might be less critical. But when the Travis builds will start to fail because of a major update in phpunit, you'll see me with a smug face: "I told you so" :p

Reply:
You're actually working against your own point. Because if Travis fails because some newly released dependency doesn't work, I want to know that. (So that we can either fix it or add a stricter version requirement)

Reply:
Ok, as you wish, we both have valid points. I removed composer.lock. I guess it boils down to the different ways one can have of managing dependencies during dev :)